center/src/main/resources/application.yml

```diff
@@ -51,14 +51,18 @@ app:
   access-token-limit: 2
   # Mail Address Format
   default-user: ${DEFAULT_USER:default@hydralab.com}
-  blob:
-    connection: ${BLOB_CONNECTION_STR}
-    fileLimitDay: 6
-    CDNUrl: ${CDN_URL:@null}
-    SASExpiryTimeFront: ${BLOB_SAS_EXPIRY_FRONT:120}
-    SASExpiryTimeAgent: ${BLOB_SAS_EXPIRY_AGENT:120}
-    SASExpiryUpdate: ${BLOB_SAS_EXPIRY_UPDATE:10}
-    timeUnit: ${TIME_UNIT:MINUTES}
+  storage:
+    type: AZURE # current available options: LOCAL, AZURE
+    blob:
+      connection: ${BLOB_CONNECTION_STR}
```

Comment:
This change (change of the connection string name) has an impact on the service onboarding/out-of-box experience, you may need to update the related wiki as well.

Reply:
For current changes, no extra/new operation is required for user of Uber image, the pass-in parameter is still BLOB_CONNECTION_STR.
I will update the README when local storage is implemented, I will illustrate all supported storage types of Hydra Lab and list the corresponding inputs then.